guestbook/README.md

```diff
@@ -4,704 +4,294 @@
-    - [Step One: Start up the redis master](#step-one-start-up-the-redis-master)
-      - [Define a Deployment](#define-a-deployment)
-      - [Define a Service](#define-a-service)
-      - [Create a Service](#create-a-service)
-      - [Finding a Service](#finding-a-service)
-        - [Environment variables](#environment-variables)
-        - [DNS service](#dns-service)
-      - [Create a Deployment](#create-a-deployment)
-      - [Optional Interlude](#optional-interlude)
-    - [Step Two: Start up the redis slave](#step-two-start-up-the-redis-slave)
-    - [Step Three: Start up the guestbook frontend](#step-three-start-up-the-guestbook-frontend)
-      - [Using 'type: LoadBalancer' for the frontend service (cloud-provider-specific)](#using-type-loadbalancer-for-the-frontend-service-cloud-provider-specific)
-    - [Step Four: Cleanup](#step-four-cleanup)
-    - [Troubleshooting](#troubleshooting)
-    - [Appendix: Accessing the guestbook site externally](#appendix-accessing-the-guestbook-site-externally)
-      - [Google Compute Engine External Load Balancer Specifics](#google-compute-engine-external-load-balancer-specifics)
+{% endcapture %}
 
-<!-- END MUNGE: GENERATED_TOC -->
+{% capture objectives %}
+* Start up a Redis master.
+* Start up Redis slaves.
+* Start up the guestbook frontend.
+* Expose and view the Frontend Service.
+* Clean up.
+{% endcapture %}
 
-The example consists of:
+{% capture prerequisites %}
 
-- A web frontend
-- A [redis](http://redis.io/) master (for storage), and a replicated set of redis 'slaves'.
+{% include task-tutorial-prereqs.md %}
+Download the following configuration files:
+1. [redis-master-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-master-deployment.yaml)
+2. [redis-master-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-master-service.yaml)
+3. [redis-slave-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-slave-deployment.yaml)
+4. [redis-slave-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-slave-service.yaml)
+5. [frontend-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/frontend-deployment.yaml)
+6. [frontend-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/frontend-service.yaml)
 
-The web frontend interacts with the redis master via javascript redis API calls.
+{% endcapture %}
 
-**Note**:  If you are running this example on a [Google Container Engine](https://cloud.google.com/container-engine/) installation, see [this Google Container Engine guestbook walkthrough](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead. The basic concepts are the same, but the walkthrough is tailored to a Container Engine setup.
+{% capture lessoncontent %}
 
-### Prerequisites
+## Start up the Redis Master
```

Comment:
nit: always use empty line after headings to prevent rendering nuances in different markdown parsers